scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -198,11 +198,11 @@ task CreateCostObservabilityTable {
       PARTITION_STRING="--time_partitioning_field call_start_timestamp --time_partitioning_type DAY"
       echo "making table $TABLE"
       echo '~{cost_observability_json}' > schema.json
-      bq --location=US mk ${PARTITION_STRING} --project_id=~{project_id} $TABLE schema.json
+      bq mk ${PARTITION_STRING} --project_id=~{project_id} $TABLE schema.json
     fi
   >>>
   runtime {
-    docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:404.0.0-alpine"
+    docker: "us.gcr.io/broad-dsde-methods/variantstore:2022-10-06-alpine"
```

Comment:
couldn't this stay `gcr.io/google.com/cloudsdktool/cloud-sdk:404.0.0-alpine` (which is smaller)?

Reply:
I thought I needed to change that one earlier, but it actually runs just fine using the smaller version of the file.  Good catch!